### PR TITLE
LPS-204352 Feature flags Playwright fixture

### PR DESCRIPTION
**Pull request by 4lejandrito**
Hey @gabrielwas, @cgmonte! I've been learning about Playwright fixtures and I applied my new knowledge to the current tests in master.

I added a feature flags fixture, and also created a base test fixture that all tests can share.

Feel free to do whatever you want with this.

**Edit:** all tests pass locally.

### CODE REVIEW

**Comment by 4lejandrito** [comment]
ci:test:sf

File: modules/test/playwright/fixtures/featureFlags.fixture.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+exports.test = test.extend({
+	_withFeatureFlags: async ({_apiHelpers}, use) => {
+		await use(async (featureFlags, code) => {
+			for (const featureFlag of featureFlags) {
+				await _apiHelpers.featureFlag.updateFeatureFlag(
+					featureFlag,
+					'true'
+				);
+			}
+			await code();
```
Comment (izaera) [inline modules/test/playwright/fixtures/featureFlags.fixture.js:17]: you need a try/finally here to make sure you disable the FF no matter what, right?

Comment (4lejandrito) [reply modules/test/playwright/fixtures/featureFlags.fixture.js]: See https://github.com/liferay-objects/liferay-portal/pull/2966/commits/778f2a752aad2e2477a1f1f61509fa41cdaeb9aa.

File: modules/test/playwright/fixtures/featureFlags.fixture.js [hunk 2]
```diff
@@ -14,12 +14,16 @@ exports.test = test.extend({
...
-			await code();
-			for (const featureFlag of featureFlags) {
-				await _apiHelpers.featureFlag.updateFeatureFlag(
-					featureFlag,
-					'false'
-				);
+			try {
+				await code();
+			}
+			finally {
```
Comment (izaera) [inline modules/test/playwright/fixtures/featureFlags.fixture.js:20]: much better :sweat_smile:

File: modules/test/playwright/fixtures/test.fixture.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {mergeTests, test as base} from '@playwright/test';
+
+import {test as apiHelpersTest} from './apiHelpers.fixture';
+import {test as featureFlagsTest} from './featureFlags.fixture';
+
+export const test = mergeTests(base, apiHelpersTest, featureFlagsTest);
```
Comment (izaera) [inline modules/test/playwright/fixtures/test.fixture.js:11]: I like this idea of having an augmented base `test` for all our tests  :+1:

**Comment by cgmonte** [comment]
I am closing this PR as there is another PR dealing with problem [here](https://github.com/izaera/liferay-portal/pull/168).

**Pull request closed without merging**